get_github_coderabbit_review/coderabbit_fetcher/cache/file_cache.py

```
@@ -0,0 +1,501 @@
+            return False
+
+        except Exception:
+            return False
+
+    def _cleanup_cache(self) -> None:
+        """Cleanup cache by removing oldest files."""
+        try:
+            # Get all cache files with their modification times
+            cache_files = []
+
+            for root, dirs, files in os.walk(self.cache_dir):
```

Ruff (0.13.2)

Loop control variable `dirs` not used within loop body

Rename unused `dirs` to `_dirs`

(B007)